### ISSUE TITLE
What is pretrained-model?

### ISSUE THREAD

**Issue by buimanhlinh96** [JDAI-CV/FaceX-Zoo#11]
Hi @wang21jun ,
Thanks for your contributor!
In your submit at https://github.com/JDAI-CV/FaceX-Zoo/tree/main/training_mode, i see you had two kind models: SOTA backbones model and SOTA heads model. What is the difference between them?
Thank you!!

**Comment by wang21jun** [comment]
SOTA backbones model : models which are trained by the SOTA backbones with the same head, i.e. MV-Softmax.
SOTA heads model : models which are trained by the SOTA heads with the same backbone, i.e. MobileFaceNet.

**Comment by buimanhlinh96** [comment]
@wang21jun I got it!
Thank you for your clear answer. You can close this issue!

**Comment by buimanhlinh96** [comment]
If I just want to train head, not re-train the backbone in my custom dataset. How can i solve it?

**Comment by vietth-bka** [comment]
> If I just want to train head, not re-train the backbone in my custom dataset. How can i solve it?

You can search keyword freezing backbone pytorch.
Specifically, you can set requires_grad = False for the whole learnable parameters of the backbone. But the easiest way is just putting the parameters of head into the optimizer but not the params of backbone.

**Comment by buimanhlinh96** [comment]
@vietth-bka thank you!
